core-services/secret-mirroring/_mapping.yaml

```diff
@@ -192,3 +192,15 @@ secrets:
   to:
     namespace: bastion-telco
     name: sshd-host-keys
+- from:
+    namespace: registry-login-secret
```

Comment:
That's not a great name for a namespace... I suppose it was only created for the purpose of holding this secret, but could the name indicate something more tied to the purpose - team name (that's ideal), project name...

Reply:
@petr-muller Updated, Does it make sense now ?

Reply:
perfect!